lib/chores/populator.rb

```diff
@@ -0,0 +1,177 @@
+require "json"
+
+class Populator
+  def initialize(model, n_records=100)
+    @model = model
+    @n_records = (n_records || 100).to_i
+  end
+
+  def run(n_records=@n_records)
+    method_name = "make_#{@model.to_s.downcase}_record".to_sym
+    unless respond_to?(method_name, true)
```

Comment:
I think of it like an abstract class. There might be a valid model called `Foo` but no populator support for it (yet). Hence the custom error message.